app/community/[id]/error.tsx

```
@@ -0,0 +1,14 @@
+"use client";
+
+import FeedbackPage from "@/app/components/feedback-page";
+
+export default function Error() {
```

Biome (2.4.6)

[error] Do not shadow the global "Error" property.

(lint/suspicious/noShadowRestrictedNames)